package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@pderas/vue2-searchbar",
+  "version": "1.1.2",
```

Comment:
I'm just curious, wasn't it here before? Shouldn't you just update `version`?

Reply:
when you update in package.json, it should automatically update here too (I'm assuming you didn't write this colin?

Reply:
I did not write this. First time messing with such things...